Documentation/project-docs/developer-guide.md

```diff
@@ -246,5 +246,23 @@ If coverage succeeds, the code coverage report will be generated automatically a
 
 Code coverage reports from the continuous integration system are available from the links on the front page of the corefx repo.
 
+### Building tests with .NET Native (Windows only)
+
+.NET Native is a technology that allows compiling IL applications down into a native executable and minimal set of native DLLs, containing all needed functionality from the .NET Framework in native format.  For CoreFX tests, .NET Native support in CoreFX is relatively early, but supported.  
+
+```cmd
+// To run a single project with the .NET Native toolchain, set the appropriate build flags:
+cd src\Microsoft.CSharp\tests
+msbuild /t:BuildAndTest /p:TestTFM=netcore50aot  /p:TestNugetRuntimeId=win10-x64-aot /p:UseDotNetNativeToolchain=true
```

Comment:
I gave this a try in the dev/api repo and it fails due to missing XUnit API dependencies:

```
S:\c1\packages\Microsoft.DotNet.TestILC\1.4.24208-prerelease\contentFiles\any\any\TestILC\ProductIlc\IlcInternals.targets(1592,5)
: warning : ILTransform_0010: Could not find assembly 'System.Diagnostics.Process, Version=4.0.0.0, Culture=neutral, PublicKeyTok
en=b03f5f7f11d50a3a' referenced by 'Microsoft.DotNet.xunit.performance.runner.cli'. [S:\c1\src\System.Net.Primitives\tests\Functi
onalTests\System.Net.Primitives.Functional.Tests.csproj]
S:\c1\packages\Microsoft.DotNet.TestILC\1.4.24208-prerelease\contentFiles\any\any\TestILC\ProductIlc\IlcInternals.targets(1592,5)
: warning : ILTransform_0010: Could not find assembly 'System.Diagnostics.Process, Version=4.0.0.0, Culture=neutral, PublicKeyTok
en=b03f5f7f11d50a3a' referenced by 'xunit.performance.run.core'. [S:\c1\src\System.Net.Primitives\tests\FunctionalTests\System.Ne
t.Primitives.Functional.Tests.csproj]
```